src/ui/public/index_patterns/index_patterns_api_client.js

```diff
@@ -0,0 +1,102 @@
+import { resolve as resolveUrl, format as formatUrl } from 'url';
+
+import { pick, mapValues } from 'lodash';
+import 'whatwg-fetch';
```

Comment:
Do we use this module anywhere else?  Can we just remove it from package.json entirely?